programs/ssl/ssl_client2.c

```diff
@@ -1226,6 +1230,11 @@ int main(int argc, char *argv[])
             if (opt.tickets < 0) {
                 goto usage;
             }
+        } else if (strcmp(p, "new_session_tickets") == 0) {
+            opt.tickets = atoi(q);
```

Comment:
Copy-pasta: I assume you mean `opt.new_session_tickets` here. (This also highlights the lack of testing. Again, not a blocker, but we'll probably want a follow-up.)

Reply:
Oops, thanks.